tests/integration/questionnaires.test.js

```diff
@@ -0,0 +1,128 @@
+const expect = require('expect');
+const app = require('../../server');
+const { populateDB, questionnairesList } = require('./seed/questionnairesSeed');
+const { getToken } = require('./seed/authenticationSeed');
+
+describe('NODE ENV', () => {
+  it('should be \'test\'', () => {
+    expect(process.env.NODE_ENV).toBe('test');
+  });
+});
+
+describe('QUESTIONNAIRES ROUTES - POST /questionnaires', () => {
+  let authToken;
+  beforeEach(populateDB);
+
+  describe('VENDOR_ADMIN', () => {
+    beforeEach(async () => {
+      authToken = await getToken('vendor_admin');
+    });
+
+    it('should create questionnaire', async () => {
+      const response = await app.inject({
+        method: 'POST',
+        url: '/questionnaires',
+        headers: { Cookie: `alenvi_token=${authToken}` },
+        payload: { title: 'test', type: 'expectations_collection' },
+      });
+
+      expect(response.statusCode).toBe(200);
+    });
+
+    it('should return 400 if no name', async () => {
```

Comment:
should return 400 if no title*, plutôt ?

Reply:
done